CHANGELOG.md

```diff
@@ -9,6 +9,35 @@ stages](https://github.com/kubernetes/community/blob/master/contributors/devel/s
 
 ## Unreleased
 
+## 0.3.0 - 2019-01-23
+
+### Added
+
+- Add documentation for OVS pipeline.
+- Add Network Policy information to monitoring CRDs for both the Agent and the Controller.
+- Add support for the IPsec ESP protocol for GRE tunnels only; it can be enabled by applying antrea-ipsec.yml instead of antrea.yml.
```

Comment:
Would you list bigger features first?

Reply:
Done, it's a bit subjective but I tried to shuffle things around to put things that are likely to matter to users first,